test/replication.test.js

```diff
@@ -1699,6 +1699,111 @@ describe('Replication / Change APIs', function() {
     });
   });
 
+  describe('Replication with chunking', function() {
+    beforeEach(function() {
+      var test = this;
+      SourceModel = this.SourceModel = PersistedModel.extend(
+        'SourceModel-' + tid,
+        { id: { id: true, type: String, defaultFn: 'guid' } },
+        { trackChanges: true, replicationChunkSize: 1 });
+
+      SourceModel.attachTo(dataSource);
+
+      TargetModel = this.TargetModel = PersistedModel.extend(
+        'TargetModel-' + tid,
+        { id: { id: true, type: String, defaultFn: 'guid' } },
+        { trackChanges: true, replicationChunkSize: 1 });
+
+      var TargetChange = TargetModel.Change;
+      TargetChange.Checkpoint = loopback.Checkpoint.extend('TargetCheckpoint');
+      TargetChange.Checkpoint.attachTo(dataSource);
+
+      TargetModel.attachTo(dataSource);
+
+      test.startingCheckpoint = -1;
+    });
+
+    describe('Model.replicate(since, targetModel, options, callback)', function() {
+      it('Replicate data using the source model with chunking', function(done) {
```

Comment:
The test name should read like a sentence, where `it` stand for the subject of "describe".

```
it('calls bulkUpdate multiple times')
```